dev/scripts/components/Emblem.js

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
 
-const Emblem = ({ type }) => {
-  const capitalType = type.charAt(0).toUpperCase() + type.slice(1);
+const Emblem = (props) => {
+  // const capitalType = type.charAt(0).toUpperCase() + type.slice(1);
 
   return (
     <div className="typeHolder">
-      <img
-        src={`../../../images/${capitalType}.png`}
-        alt={`an emblem of the type ${capitalType}`}
-        title={`${capitalType} Type Icon`}
-      />
+      <label htmlFor={`${props.type}`}>
+        <img 
+          src={`../../../images/${props.type}.png`}
```

Comment:
T.T
yooo I'm not sure how the file path still works. All our emblem `.png` files start with a capital letter. eg. the path is now /colorless.png, but the file is called /Colorless.png.
To avoid any potential errors down the road, maybe we can normalize the image names to use lowercase/camelcase, to match the format of the emblemType array.